turbinia/e2e/teardown-gce.sh

```diff
@@ -0,0 +1,36 @@
+#/bin/bash
+# This script will cleanup and teardown the Turbinia Terraform setup after
+# the Turbinia e2e test is finished.
+echo "Tear down Terraform Turbinia infrastructure."
+
+DEVSHELL_PROJECT_ID=`gcloud config list --format 'value(core.project)'`
+SA_MEMBER="terraform@$DEVSHELL_PROJECT_ID.iam.gserviceaccount.com"
+
+cd ./forseti-security/contrib/incident-response/infrastructure/
+terraform destroy --target=module.turbinia -var gcp_project=$DEVSHELL_PROJECT_ID -auto-approve
+
+# Remove test and evidence disks (test-disk2 and evidence* disks)
+ZONE="us-central1-a"
+gcloud -q compute disks delete test-disk2 --zone=$ZONE
+for d in `gcloud compute disks list --uri`
+do
+  DISK=`echo $d | cut -d '/' -f 11`
+  ZONE=`echo $d | cut -d '/' -f 9`
+  if [[ $DISK == "evidence"* ]]
+  then
+    echo "Deleting $DISK in $ZONE"
+    gcloud -q compute disks delete $DISK --zone=$ZONE
+  fi
+done
+
+# TODO(rbdebeer) Debug and fix remove-iam-policy-binding calls.
```

Comment:
Is this something terraform can do for us (remove the IAM permissions for the SA)?  IIRC there are some options there for which resources we destroy, so maybe we just need to flip a bit somewhere.

Reply:
The service account and IAM permission are not created by Terraform but by the deploy.sh script so we need to remove them manually. I'll figure out a way to do this or I might just leave the serviceaccount/IAM permission but only remove the exported keys (Terraform needs those) as there is a limit to how many keys a service account can have attached to it. It will fail if too many requests are made to create/export keys.